src/components/List/ListItem/ListItem.scss

```diff
@@ -0,0 +1,27 @@
+.ListItem {
+    &__container {
+        display: flex;
+        align-items: baseline;
+        padding: 0 var(--spacing-3x);
+    }
+
+    &:hover {
+        background-color: var(--color-brand-10);
+    }
+
+    &--isSelected {
+        background-color: var(--color-brand-10);
+    }
+
+    &--isDividedList:not(:last-child) {
```

Comment:
`:not` [doesn't have much support](https://caniuse.com/#feat=css-not-sel-list)

I'd still go for using adjacent sibling selectors (+)

Reply:
you link is incorrect. It is about having a list of selector to be ignored. Simple `:not()` is [well supported](https://caniuse.com/#feat=css-sel3).